lib/job_aggregator.rb

```diff
@@ -2,16 +2,29 @@ class JobAggregator
   def initialize
     @ignore_link_text ||= []
     @url ||= ''
+    @num_pages = ARGV[0] ? ARGV[0].to_i : 1
+
   end
 
-  attr_reader :url, :ignore_link_text
+  attr_reader :ignore_link_text, :num_pages
+  attr_accessor :url
 end
 
 class MadeInNyc < JobAggregator
   def initialize
-    @url = "http://nytm.org/made-in-nyc"
+    @url = "https://nytm.org/made?list=true"
     @ignore_link_text = ['(hiring)']
 
     super
   end
+
+  def paged_urls(pages)
+    urls = []
+    (1..pages).each do |page|
+      urls << @url + "&page=#{page}"
+    end
+   
+    urls
+  end
+
 end
```

Comment:
Yeah, looks good!  ```scour_aggregator``` needs to support aggregators that don't have paged URLS, as well